k8s-daemonset/k8s/linkerd-rbac-beta.yaml

```diff
@@ -0,0 +1,48 @@
+# RBAC configs for linkerd
+---
+# allow linkerd/namerd permissions to enable service discovery
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: endpoints-reader
+rules:
+  - apiGroups: [""] # "" indicates the core API group
+    resources: ["endpoints", "services"]
```

Comment:
yeahhh I see. I think I'll add pods and then also add a comment explaining that pods are only required for "*-legacy.yml" configs :) That way this rbac config will work for all the examples, but people who want to be more restrictive can modify this.